lease/lessor.go

```diff
@@ -651,9 +810,19 @@ func (l *Lease) TTL() int64 {
 	return l.ttl
 }
 
+// RemainingTTL returns the last checkpointed remaining TTL of the lease.
+// TODO(jpbetz): do not expose this utility method
+func (l *Lease) RemainingTTL() int64 {
+	if l.remainingTTL > 0 {
+		return l.remainingTTL
+	} else {
+		return l.ttl
+	}
```

Comment:
no need `else`? just return `return l.ttl` following Go idioms? :)

Reply:
sounds good! This is one of the hardest idoms to unlearn from other languages that do the opposite :)